uk.co.compucorp.civicrm.hrcontactactionsmenu/scss/components/_contact-actions.scss

```diff
@@ -0,0 +1,54 @@
+.crm_contact-actions {
+  background: $gray-light;
+  width: 1065px !important;
+
+  &::before {
+    background: $gray-light !important;
+    border-bottom: none !important;
+    height: 20px !important;
+    left: 0 !important;  
+    top: -15px !important;
+    width: 93px !important;
```

Comment:
why all these `!important`s ?

Reply:
These `!important` s are used to override the styles defined for `:before` pseudo element. Which was like below previously:
<img width="829" alt="35151711-7f60ef7a-fd47-11e7-8202-e00e49b1a6c0" src="https://user-images.githubusercontent.com/6307362/35320030-6d0d1fc6-010a-11e8-9af9-ec49a98bec43.png">

And by overriding these styles, this PR makes the design like following to match the design:
<img width="1328" alt="screen shot 2018-01-22 at 1 40 31 pm" src="https://user-images.githubusercontent.com/6307362/35320105-c4167c2c-010a-11e8-967f-8268e247c01f.png">